activities/zio-flow-twilio/shared/src/main/scala/zio/flow/activities/twilio/Direction.scala

```diff
@@ -0,0 +1,18 @@
+package zio.flow.activities.twilio
+
+import zio.flow.Remote
+import zio.schema.DeriveSchema
+
+sealed trait Direction
+object Direction {
+  case object inbound          extends Direction
```

Comment:
We cannot specify the "serialized name" of constructors to be used in json codec so we cannot follow Scala naming conventions

Reply:
There's a ticket in ZIO Schema for that.